Godeps/Godeps.json

```diff
@@ -63,14 +58,29 @@
 			"Rev": "b965b613227fddccbfffe13eae360ed3fa822f8d"
 		},
 		{
+			"ImportPath": "github.com/codegangsta/inject",
```

Comment:
Why introduce codegangsta/inject package here? I couldn't find go-etcd has depency on this one.